portfolio/src/main/webapp/script.js

```diff
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
+
+ /* Adds a random greeting to the page.
```

Comment:
Remove this function if it's commented out, do not check in death pieces of code.

Reply:
Solved